src/index.js

```diff
@@ -121,7 +121,8 @@ class MongoOplogReader {
         .then(info => {
           const usedMem = info.match(/used_memory:(\d*)/)[1];
           const totalMem = info.match(/total_system_memory:(\d*)/)[1];
-          const memUsage = usedMem / totalMem;
+          const maxMem = info.match(/maxmemory:(\d*)/)[1];
+          const memUsage = usedMem / (maxMem || totalMem);
```

Comment:
What are the chances that both `maxMem` and `totalMem` are mis-reading and output 0?

Reply:
totalMem will always be > 0